tests/conftest.py

```diff
@@ -55,29 +55,12 @@ def check_auth_app(session_dir):
     return SessionMiddleware(check_auth, auto=1, data_dir=session_dir)
 
 @pytest.fixture(scope="module")
-def authorized_session_id(check_auth_app):
-    # FIXME: I shouldn't have to do this, but the store doesn't get initialized until the first request
+def authorized_session_id(pcic_data_portal):
```

Comment:
This is soooo much simpler and less prone to breakage. Thank goodness. The old way was a mess; good work.